### ISSUE TITLE
Line wrapping breaks Pango markeup

### ISSUE THREAD

**Issue by gwyndaf** [ChordPro/chordpro#180]
A long line that includes Pango markup, which wraps such that start <> and end </> tags fall either side of wrap point 1) gives error 2) doesn't apply markup to 'wrapped' portion.

## Describe the bug
Processing attached Line_wrap_Pango_test file produces errors:
```
Markup error: "\  <i>(¿sabes qué ropan me"
  Unclosed markup: <i>
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF/Writer.pm line 201.
Markup error: "\  <i>(¿sabes qué ropan"
  Unclosed markup: <i>
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF/Writer.pm line 201.
Markup error: "\  <i>(¿sabes qué ropan"
  Unclosed markup: <i>
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF/Writer.pm line 201.
Markup error: "me quedan?)</i>"
  Closing <i> but no markup is open
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF/Writer.pm line 201.
Markup error: "me quedan?)</i>"
  Closing <i> but no markup is open
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF/Writer.pm line 201.
Markup error: "Padding to make the lines longer. Don’t you know which clothes even |fit me?  \\\ \\\\  \  <i>(¿sabes qué ropan"
  Unclosed markup: <i>
Markup error: "me quedan?)</i>"
  Closing <i> but no markup is open
 at /usr/local/share/perl/5.32.1/App/Music/ChordPro/Output/PDF.pm line 1987.
Markup error: "me quedan?)</i>"
  Closing <i> but no markup is open
```

### To Reproduce
1. Process attached  with `chordpro --nouserconfig Line_Wrap+Pango_test.cho`
2. Observe error messages (as above)
3. Observe output PDF: main line respects markup, wrapped portion doesn't

### Expected behavior
Expect no error and markup to apply to relevant portion of line, including wrapped part.

### Attachments
1. Source file: [Line_Wrap+Pango_test.cho.txt](https://github.com/ChordPro/chordpro/files/7189715/Line_Wrap%2BPango_test.cho.txt)
2. Sample output: [Line_Wrap+Pango_test.pdf](https://github.com/ChordPro/chordpro/files/7189716/Line_Wrap%2BPango_test.pdf)

## System information
 - OS: Linux (Debian bullseye)
 - Where did you get ChordPro from? CPAN
 - ChordPro version: 5.982
 - Perl version: 5.32.1

## Additional context
I'm guessing this arises because Chordpro processes text on a line-by-line basis, and the source line gets split _before_ processing Pango markup.

It's a very rare occurrence for me, as song lines aren't usually so long and, if they are, I split manually. Still thought it worth flagging, though.

**Comment by sciurius** [comment]
Yes, this is a known problem, and hard to solve. Sorry for that.

Leave the issue open so it won't get forgotten.
